### PR TITLE
Adjustflow

### PR DESCRIPTION
**Pull request by Decodetalkers**

### CODE REVIEW

**Comment by Decodetalkers** [comment]
This pr will also reduce the times of dispatch , and the counts of clone

File: libwayshot/src/lib.rs [hunk 1]
```diff
@@ -77,6 +77,7 @@ struct IntersectingOutput {
 pub struct WayshotConnection {
     pub conn: Connection,
     pub globals: GlobalList,
+    output_infos: Vec<OutputInfo>,
```
Comment (Shinyzenith) [inline libwayshot/src/lib.rs:80]: Hm would we benefit from changing this to pub?

File: libwayshot/src/lib.rs [hunk 2]
```diff
@@ -387,10 +403,26 @@ impl WayshotConnection {
...
+        output_info: &OutputInfo,
+        cursor_overlay: bool,
+    ) -> Result<RgbaImage> {
+        let frame_copy = self.capture_output_frame(
+            cursor_overlay as i32,
+            &output_info.wl_output,
+            output_info.transform,
+            None,
+        )?;
+        let dynamicimage: DynamicImage = (&frame_copy).try_into()?;
```
Comment (Shinyzenith) [inline libwayshot/src/lib.rs:418]:
```suggestion
        let dynamic_image: DynamicImage = (&frame_copy).try_into()?;
```

Style nitpick

**Review by Shinyzenith — APPROVED** [review]

**Comment by Shinyzenith** [comment]
Apart from these minute changes everything seems fine. You can merge once fixed.

File: libwayshot/src/lib.rs (hunk 1 above)
Comment (Decodetalkers) [reply libwayshot/src/lib.rs:80]: I think it should not be changed by user..

**Comment by Decodetalkers** [comment]
then I can finally works on 56, and 56 is merged, I will work on xdg-desktop-portal

File: libwayshot/src/lib.rs (hunk 1 above)
Comment (Shinyzenith) [reply libwayshot/src/lib.rs:80]: Ok that makes sense

**Review by Shinyzenith — APPROVED** [review]

**Pull request merged**